src/main/java/mchorse/metamorph/api/morphs/AbstractMorph.java

```diff
@@ -224,8 +224,8 @@ else if (health > IMorphing.REASONABLE_HEALTH_VALUE)
         // We need to retrieve the max health of the target after modifiers are
         // applied
         // to get a sensible value
-        float proportionalHealth = Math.round(target.getMaxHealth() * ratio);
-        target.setHealth(proportionalHealth <= 0 ? 1 : proportionalHealth);
+        float proportionalHealth = target.getMaxHealth() * ratio;
+        target.setHealth(proportionalHealth <= 0.0F ? Float.MIN_VALUE : proportionalHealth);
```

Comment:
Another rare case. I think I wrote this code back when I thought health values could only be whole numbers, which was wrong.